### PR TITLE
fix: dnd5e mapping improvements

### PR DESCRIPTION
**Pull request by cogitocodergh**
- translate and organize proficiencies, respect semicolon in custom lists (#20)
- use derivedDamage strings in attacks (#19)
- strip html chars from personality traits, ideals, bonds, and flaws (compatibility with tidy5e)
- include to-hit bonus in attacks
- star indicates additional damage from attack (more than one damage part)
- equipment sheet includes only weapons, equipment, and tools; containers and consumables moved to treasure for space management
- spells removed from equipment list

### CODE REVIEW

File: mappings/dnd5e.mapping
Comment (arcanistzed) [inline mappings/dnd5e.mapping:48]: The Weapon Damage fields are labelled `DAMAGE/TYPE` on the PDF, so perhaps it would be appropriate to also add the damage type to the output?

More importantly, you should handle for when the result of this expression is undefined by adding optional chaining in your return statement: `return dda?.[0]` and outputting an empty string if it's undefined: `?? ""`.

**Review by arcanistzed — CHANGES_REQUESTED** [review]
Thanks again! I'll be ready to merge this once the issue with the undefined return value is resolved for the Weapon Damage fields.

**Comment by arcanistzed** [comment]
I pushed the necessary fixes to the [dnd5e-mapping-improvements](https://github.com/arcanistzed/pdf-sheet/tree/dnd5e-mapping-improvements) branch.

File: mappings/dnd5e.mapping
Comment (cogitocodergh) [reply mappings/dnd5e.mapping]: Done

File: mappings/dnd5e.mapping
Comment (cogitocodergh) [reply mappings/dnd5e.mapping]: Fixed

File: mappings/dnd5e.mapping
Comment (cogitocodergh) [reply mappings/dnd5e.mapping]: Added dmg type and fixed null/undef handling

**Comment by cogitocodergh** [comment]
I merged the commits from your upstream feature branch, although I redid most of that stuff to address the comments you'd made already.  I think I have the case where damage shows as undefined, but if you have a scenario where it occurs let me know and I can stomp it out.  Thanks!

File: mappings/dnd5e.mapping
Comment (arcanistzed) [inline mappings/dnd5e.mapping:120]: This is super minor and unimportant, but I suggest putting a semicolon at the end of the line here.

Comment (arcanistzed) [inline mappings/dnd5e.mapping:120]: This is super minor and unimportant, but I suggest putting a semicolon at the end of the line here.

Comment (cogitocodergh) [reply mappings/dnd5e.mapping]: Done -- also see a revision to how movement is handled.  Please squash on merge, or allow me to do so on my feature branch before merging.

**Review by arcanistzed — APPROVED** [review]

**Pull request merged**
